main/main.c

```diff
@@ -186,6 +178,12 @@ void ex2_init(void *pvParameters) {
 
     init_software();
 
+    /* LEOP */
```

Comment:
Why shouldn't leop be the first thing in ex2_init? It would be ideal if there was as little code running as possible during LEOP to mitigate the risk of failure or error.

Reply:
We need at minimum the logger and DFGM service running, but having everything else running should be fine. If we bump the priority of the LEOP task it would be even better

Reply:
Ah just kidding, the task that handles LEOP, ex2_init, is the maximum priority

Reply:
Yes, definitely start logger before.

Regardless of this task's priority, do you see any  risk in running all the other tasks concurrently with leop? It just seems like a small amount of unnecessary risk that can be eliminated by starting them after leop is done

Reply:
Id say dfgm doesnt need to start before leop until there's a stronger resolution on that slack discussion

Reply:
Yes, we'll test it many times before we deploy code, BUT, there is always a chance of a sneaky memory leak affecting leop so making this bulletproof seems worth doing. I see no downside or risk to making the logger the only thing that starts before leop. I talked w/ Steven about this to see if there were any factors I was overlooking and he sees it the same way.

Reply:
If we implement the change where we use a separate log file for LEOP this all becomes irrelevant anyway.

Reply:
How does it all become irrelevant? I don't understand because the risk of memory leaks affecting leop increases with the amount of code running concurrently w/ leop.

Reply:
I still haven't seen any justification for why it's _advantageous_ to keep all these inits before leop. Regardless if it becomes irrelevant in the future, it's relevant now.